wallet/src/Pos/Wallet/Web/Pending/Types.hs

```diff
@@ -89,7 +89,7 @@ data PtxCondition
                                       --   (this constructor is at bottom for
                                       --   the sake of SafeCopy instance's
                                       --   backward-compatibility)
-    deriving (Eq, Show)
+    deriving (Eq, Ord, Show)
```

Comment:
Are we sure this `Ord` instance is what we want? I appreciate that's probably needed by our lovely `ixset-typed` filtering to work, but the "stock" derivation will derive ordering based on the order these type constructors are listed. What does it mean for a `PtxCondition` to be comparable? Is it really true that `PtxApplying a` >= `PtxCreating ptxPool` ? I don't have a good answer for you at this stage yet as I can see how we do want to be able to filter info out from the REST API, but it doesn't make sense for all resources to be `ord`erable. Let's tackle this in an another PR, for now let's disregard it (but we should track this into a ticket nevertheless, imho).

Reply:
I put it on there because `Transaction` has an `Ord` instance, and it demanded one for this. I'll try removing it and see what happens.

Reply:
Yup, we need an `Ord` instance on `Transaction` in order to put them in an `IxSet`.

Reply:
this is another point in favor of moving to SQL instead of ixset. Though the ordering is entirely arbitrary and only used for constructing the set. Could instead use a hashed set, but that would likely require a rewrite of ixset library, and then you're just trading `Ord a` for `Hashable a, Eq a`.

Reply:
Yes, I think the reason you need `Ord` is because under the hood (as you probably have noticed) `ixset` relies on normal sets, thus you need to be able to compare the data.

> this is another point in favor of moving to SQL instead of ixset.

You don't give up easily, do you? 😁 Jokes aside, I'm not particularly attached to `ixset`s and I'm happy to review our options (when @dcoutts is back, hehe). All the "scouting" work we are doing I think is extremely important though, as it means @edsko will be able to take more informative decisions as part of https://iohk.myjetbrains.com/youtrack/issue/CSL-2105 😉